### DISCUSSION TITLE
Linkedin Attacks

### DISCUSSION THREAD

**Discussion by daniel-burlacu** [community/community#146641] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

How you can identify Linkedin attacks ? Well, check out the text, as it is trying to push you to pull the repo, it is urgent and so on, here is a sample : 
" Our CTO just reviewed your profile and he is very interested in your experience and we are gonna move to next step of interview.

We are planning to update current MVP v1 to v2 (currently, our v1 project does not have any smart contracts)

As you know, most of people on Linkedin have good reviews and experiences so we think skill test is simple and efficient way for prove developer's experience.

So our CTO prepared simple and basic test process for it.
It will takes 10 ~ 15mins for finish it.

--- Here is a uploaded test project - 
 When you open the project, you can see "Buy 1 NFT to gain access" button in the first page
 In other side, you can see balanceOf method in the following contract:
 
 Here, you should get a response that comes out when pass the following address in this read function:
 Wallet address: 0x248Dd3836E2A8B56279C04addC2D11F3c2497836
 After getting response, exchange the title of "Buy 1 NFT to gain access" button with it ---

Our CTO has very tight schedule so we want you to do this coding challenge asap and let us know.
Then we will schedule a quick call with our CTO "

Message sent from: 

Another message which I have received is from a Colin  Doran : 

Both linkedin profiles looked very legit to me.

> URL's hidden by staff

**Comment by shinybrightstar** [comment]
Hi @daniel-burlacu ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
